hackathon_site/hardware/views.py

```diff
@@ -52,13 +52,18 @@ def get(self, request, *args, **kwargs):
 
 
 class OrderListView(generics.ListAPIView):
-    queryset = Order.objects.all()
+    queryset = Order.objects.all().select_related("team")
```

Comment:
Still need to prefetch the related hardware:

```python
queryset = ( 
    Order.objects.all() .select_related("team") .prefetch_related("hardware", "hardware__categories") 
)
```

Reply:
Added a comment